aiogram/dispatcher/filters/builtin.py

```diff
@@ -105,7 +120,7 @@ async def check_command(message: types.Message, commands, prefixes, ignore_case=
         if (command.lower() if ignore_case else command) not in commands:
             return False
 
-        return {'command': Command.CommandObj(command=command, prefix=prefix, mention=mention)}
```

Comment:
Why replaced by directly class usage instead of cls?

Reply:
Looks better :) 
Because `classmethod` allows easy renaming and other refactroings.